wd_helper.au3

```diff
@@ -798,140 +786,158 @@ Func _WD_FrameList($sSession, $bReturnAsArray = True)
 			$a_Result[$i][$_WD_FRAMELIST_Relative] = StringRegExpReplace($a_Result[$i][$_WD_FRAMELIST_Absolute], '\A' & $sStartLocation & '\/?', '')
 		Next
 
-		$iFrameCount = UBound($a_Result, $UBOUND_ROWS)
 		; select desired DataType for the $vResult - usually string is option for testing and asking support
 		If $bReturnAsArray Then
 			$vResult = $a_Result
 		Else
 			$vResult = _ArrayToString($a_Result)
 			If @error Then
 				$iErr = $_WD_ERROR_RetValue
-				$sMessage &= ' ArrayToString conversion failed'
+				$sMessage &= 'ArrayToString conversion failed. '
 				$vResult = ''
```

Comment:
$sMessage should be empty at this point so no need to perform concatenation, correct?

Reply:
Yes. But I wanted to have the same `design pattern` within function.

Reply:
That explanation doesn't make sense. Why would you concatenate unnecessarily just because you can and it "looks nice"?

Reply:
Ok.
Please take a look: [$sMessage &=](https://github.com/Danp2/au3WebDriver/pull/434/commits/fd2530dc28f76d79aaeae1b7c326f92a0101431a)